index.html

```diff
@@ -958,13 +961,14 @@ <h3>Public authority identity credentials (eIDAS)</h3>
         Authentication and Trust Services regulation
         (<a href="https://en.wikipedia.org/wiki/EIDAS">eIDAS</a>), which standardizes
         attributes and trust services in all E.U. member states.</p>
-      <p>With her ESSIF identity, Eva can apply to the Spanish university. The university
+	    <p>With her <abbr title="European Self-Sovereign Identity Framework">ESSIF</abbr> identity, Eva can apply to the Spanish university. The university
         in turn issues further digital credentials to Eva's wallet, such as a student
-        record number, or - after successful completion of the program - a digital diploma.
+        record number, or &mdash; after successful completion of the program &mdash; a digital diploma.
         In her future life, Eva can continue to enrich and use her ESSIF identity in many
         ways, for example when applying for a job at a company in Finland, for registering
         her own business in Romania, for opening a bank account, or for applying for E.U.
-        grants. Since her ESSIF identity includes credentials from eIDAS-compliant public
+        grants. Since her <abbr title="European Self-Sovereign Identity Framework">ESSIF</abbr> identity includes credentials from <abbr title="Electronic IDentification,
```

Comment:
```suggestion
        grants. Since her <abbr title="European Self-Sovereign Identity Framework">ESSIF</abbr> 
        identity includes credentials from <abbr title="Electronic IDentification,
```